control_plane/src/lib.rs

```diff
@@ -31,3 +32,14 @@ pub fn read_pidfile(pidfile: &Path) -> Result<i32> {
     }
     Ok(pid)
 }
+
+fn fill_rust_env_vars(cmd: &mut Command) -> &mut Command {
+    let cmd = cmd.env_clear().env("RUST_BACKTRACE", "1");
+
+    const RUST_LOG_KEY: &str = "RUST_LOG";
+    if let Ok(rust_log_value) = std::env::var(RUST_LOG_KEY) {
```

Comment:
What does RUST_LOG_KEY do?

Would it make sense to pass through all env variables with `RUST_*` prefix?

Reply:
> What does RUST_LOG_KEY do?

That's just a variable to hold the `RUST_LOG` value to use it twice on the next lines. `const` forces it to BE_NAMED_THIS_WAY , so might be a bit confusing. I can define it as `let rust_log_key = ...`, would that reduce the confusion?

> Would it make sense to pass through all env variables with RUST_* prefix?

I'm not aware of any others that start with such prefix, actually. 
Seems that the `RUST_LOG` one emerged from `env_logger` crate and still isn't a standard really, since has to be supported in every logging backend separately. For instance, the `slog` one we've used before `tracing` did not support such variable.

Lastly, the only documentation about any rust-related env vars I was able to find is about `cargo`:
https://doc.rust-lang.org/cargo/reference/environment-variables.html

So, I'd rather keep the `RUST_LOG` one only for now.